python/python/res/simulator/batch_simulator.py

```diff
@@ -3,6 +3,13 @@
 from res.enkf import ResConfig, ErtRunContext, EnKFMain, EnkfConfigNode, EnkfNode, NodeId
 from .batch_simulator_context import BatchContext
 
+def _slug(entity):
+    entity = ' '.join(str(entity).strip().split())
+    for i in range(len(entity)):
```

Comment:
Couldn't the for-loop be replaced by something along the lines of:
`return map(lambda x : x if x.isalnum() else '_', entity)`?
Possibly a bit more Pythonic?